jenkins/kola/packet.sh

```diff
@@ -35,7 +35,14 @@ fi
 # so we override the `PACKET_REGION` to `Dallas` since it's available in this region.
 # We do not override `PACKET_REGION` for both board on top level because we need to keep proximity
 # for PXE booting.
```

Comment:
Maybe we can also keep the previous comment ?

Reply:
I did not delete previous comments..?

Reply:
Haha indeed - I don't know why I thought they were gone.